public/Lab-Files/Lesson03/Assignments/blog-page.css

```diff
@@ -0,0 +1,65 @@
+body {
+    background-color: #0059b3;
+}
+
+h1 {
+    font-family: "Arial Black", Gadget, sans-serif;
```

Comment:
If you add ```font-family: "Arial Black", Gadget, sans-serif;``` to the ```body```, it will apply to everything and you won't have to add it to each element separately!